docs/install/kubernetes/azure.md

```diff
@@ -0,0 +1,54 @@
+---
+navTitle: Azure AKS Installation
+meta:
+   description: Learn how to install FlowFuse on AWS EKS with setup details for EKS, Nginx Ingress, AWS SES, and RDS PostgreSQL integration.
+   tags:
+      - flowfuse
+      - nodered
+      - azure
+      - aks
+      - nginx ingress
+      - helm
+      - kubernetes
+---
+
+# Azure AKS Specific details
+
+This document includes details for installing FlowFuse on Azure AKS
+
+## Nginx Ingress
+
+We recommend using the <a href="https://kubernetes.github.io/ingress-nginx/" target="_blank">Nginx Ingress controller</a> as this is the one we test with and we have run into limits on the number of Instances with other Ingress Controllers. 
+
+```bash
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm repo update
+helm --kubeconfig=./k8s-flowforge-kubeconfig.yaml install nginx-ingress \
+  ingress-nginx/ingress-nginx --namespace ingress-nginx \
+  --create-namespace \
+  --set controller.publishService.enabled=true \
+  --set controller.ingressClassResource.default=true \
+  --set controller.config.proxy-body-size="0" 
+  --wait
+
+```
+
+The `controller.config.proxy-body-size="0"` removes the `1m` default payload limit 
+from the nginx ingress proxy. You can change this to say `5m` which will match the 
+Node-RED default value.
+
+
+## Persistent Storage
+
+If making use of the Persistent Storage feature you will need to provide a StorageClass that support MultiPod mounting.
+
+e.g. Azure Disks backed Persistent Volumes
```

Comment:
Azure Disks are available for single node only. Azure Files is the solution for shared storage accessible by multiple pods at the same time.